drivers/bridge/setup_device.go

```diff
@@ -63,7 +63,7 @@ func setupDefaultSysctl(config *networkConfiguration, i *bridgeInterface) error
 		return nil
 	}
 	if err := ioutil.WriteFile(sysPath, []byte{'0', '\n'}, 0644); err != nil {
-		return fmt.Errorf("libnetwork: Unable to disable IPv6 router advertisement: %v", err)
+		logrus.WithError(err).Info("unable to disable IPv6 router advertisement")
```

Comment:
I'd prefer to see this as a Warn rather than Info.  If we switch this patch to use https://github.com/moby/libnetwork/blob/master/osl/kernel/knobs_linux.go as suggested by @arkodg, failure to write will get logged as Error.